### PR TITLE
[webgpu] Add minimum

### PR DESCRIPTION
**Pull request by axinging**
Currently, NAN is not supported in WebGPU backend:
https://gpuweb.github.io/gpuweb/wgsl.html#floating-point-evaluation.

TODO: https://github.com/tensorflow/tfjs/issues/4734

FEATURE

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4639)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @haoyunfeix @xhcao, PTAL
minimum is used in bodypix.

File: tfjs-backend-webgpu/src/kernels/binary_ops.ts [hunk 1]
```diff
@@ -98,26 +99,21 @@ export function getBinaryOpString(
...
+      // TODO (xing.xu@intel.com): Currently, NaN is not supported in WebGPU
+      // backend.
+      // https://github.com/tensorflow/tfjs/issues/4734
       return useVec4 ? `
-      vec4 result = vec4(max(a, b));
-      vec4 isNaN = min(vec4(isnan(a)) + vec4(isnan(b)), vec4(1.0));
-      ` + CHECK_NAN_SNIPPET +
-              `
-      return result;
+      return vec4(max(a, b));
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/binary_ops.ts:106]: I think you can directly return `max(a, b)` and don't need to distinguish float and vec4. Similar for MIN.

Comment (axinging) [reply tfjs-backend-webgpu/src/kernels/binary_ops.ts]: Thanks @qjia7, updated, PTAL

**Comment by qjia7** [comment]
LGTM
+@lina128 @kainino0x take a look. Thanks.

**Review by kainino0x — APPROVED** [review]

**Pull request merged**